.github/workflows/ci.yml

```diff
@@ -34,14 +34,14 @@ jobs:
         run: npm ci
 
       - name: Build
-        run: npm run build --if-present
+        run: npm run build:dev --if-present
```

Comment:
we can stay with the current commands and just add the others for prod, something Like build:prod, test:prod and so on. In this way we simplify script naming for developer that are used much more often.

Reply:
And we also do not need to update the contributing.md